### ISSUE TITLE
 mapi_exp ortchanges_config() failed: 0xFFFFFFFF8004010F - code: 12

### ISSUE THREAD

**Issue by peterm888** [TosoBoso/Kopano4s#3]
I loaded v1.04 from cph.  Upgrade from prior version.
This error was happening on email folder refresh for one or two accounts from about 10 (home use!).  Affects "inbox" but not calendar, contacts, other email folders.
However, by deleting a few hundred emails, problem goes away.  This has happened a number of times, even on old version (0.9 I think).  Is there a space limit somewhere being exhausted?

**Comment by TosoBoso** [comment]
No space limits known to be exhausted. As your issues appear to be in Kopano or Z-Push and you would need to outline if it is an active sync issue or in web-/deskapp. Please raise it in forum.kopano.io outlining what release of Kopano and Z-Push you are using which is indicated by the k4s Docker image suffix. Assuming you are using Community edition which is based on beta nightly builds (Docker image prefix C) the advise is to switch to Stable Edition e.g default (Docker image prefix D). 
In short no bug / issue to track here but better suited in forum and once traced down issue can be raised to Kopano / Z-Push / Kopano4S for resolution. Closing issue as of now. -TosoBoso
